base/shell/explorer/taskswnd.cpp

```diff
@@ -1533,6 +1533,10 @@ class CTaskSwitchWnd :
             break;
 
         case HSHELL_ACTIVATESHELLWINDOW:
+            ::SwitchToThisWindow(m_Tray->GetHWND(), TRUE);
```

Comment:
Two naive questions:
- What would happen if FALSE was used here instead?
- If the window was minimized, is the HSHELL_ACTIVATESHELLWINDOW intended also to restore the explorer window to its original size? If so, do these two functions do that? Or should a ShowWindow call also be needed?

Reply:
FALSE means Alt+Esc that is rarely used.